jekyll/assets/css/mobile.scss

```diff
@@ -34,20 +34,9 @@
 		left: 240px;
 	}
 
-nav.sidebar{
-	position: fixed;
-	top: 70px;
-	bottom: 0;
-	left: -240px;
-	width: 240px;
-	color: #000000;
-	overflow-y: auto;
-	transition: left 0.3s ease-in-out;
-	z-index: 10;
+nav.sidebar.open{
```

Comment:
This can probably also be cleaned up, no?

Reply:
Think it might be in use [here](https://github.com/circleci/circleci-docs/blob/49307c72e8bfe0d32cfd2fe3f36f675ba01497b9/jekyll/assets/js/main.js#L263)?

Reply:
It just seemed weird to remove the base rule for `nav.sidebar` but leave the `nav.sidebar.open` rule in place. Does that element still exist on the pages?

Reply:
I think it can be cut - so I'll remove it. I think a lot of the mobile.scss file can probably be cleaned up in a future PR as it looks like `#sidebar` isn't being used anywhere.